tests/subsys/pm/device_runtime_api/src/main.c

```diff
@@ -236,15 +236,12 @@ static void test_unsupported(void)
 	zassert_equal(pm_device_runtime_put(dev), -ENOTSUP, "");
 }
 
-void test_main(void)
+void *device_runtime_api_setup(void)
```

Comment:
why this function is returning a pointer ? Shouldn't it be a `void` function ... it always return NULL.

Reply:
these lines can answer you question: 

ZTEST_SUITE calls the struct ztest_suite_node, then setup defintion, please check. 

- https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/testsuite/ztest/include/zephyr/ztest_test_new.h#L202-L213
- https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/testsuite/ztest/include/zephyr/ztest_test_new.h#L161-L182
- https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/testsuite/ztest/include/zephyr/ztest_test_new.h#L125

Reply:
@ceolin please see the comment, and review the PR again.

Reply:
yep, thanks for clarifying it.